index.html

```diff
@@ -5,9 +5,45 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap"
+      rel="stylesheet"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
     <title>My Portfolio</title>
   </head>
   <body>
-    <h1>Billy's Portfolio</h1>
+    <div class="container">
+      <div class="toolbar">
+        <p class="toolbar-text">John Doe</p>
+        <a href="#" class="menu"><img src="Assets/ham.png" alt="" /></a>
+      </div>
+      <div class="headline">
```

Comment:
- [ ] Please use the semantic `<section>` tag here because it is more meaningful and the standard way to structure sections instead of the generic div tag 😃 👍